src/qibocal/auto/runcard.py

```diff
@@ -47,21 +44,9 @@ class Runcard:
     """Structure of an execution runcard."""
 
     actions: list[Action]
-    qubits: Union[list[QubitId], list[tuple[QubitId, QubitId]]]
+    qubits: Union[list[QubitId], list[tuple[QubitId, QubitId]]] = None
     backend: str = "qibolab"
     platform: str = "dummy"
```

Comment:
Regarding the default (used when the platform is not specified, either through cli or the runcard), we could also consider using the `"QIBO_PLATFORM"` environment variable, particularly if @scarrazza is planning to use this for slurm.

One way would be to set
```suggestion
    platform: Optional[str] = None
```
and let qibo find the platform from the environment variable (this may need some fix from the qibo side).

Another way would be
```suggestion
    platform: str = os.environ("QIBO_PLATFORM", "dummy")
```
which will use the variable if exists, otherwise falls back to dummy. This is closer to the current behavior.

If we do this we will also be able to submit the same job/runcard on different queues and qibocal will know the proper platform based on the slurm (assuming it is properly set and working).

Reply:
It makes more sense to have `QIBO_PLATFORM` instead of `dummy`.
Is the prologue now working as it should @scarrazza?
If so I believe that I can commit @stavros11 's suggestion.

Reply:
Yes, expect for queues based on the same virtual node.